pkgs/applications/audio/espeak-ng/default.nix

```diff
@@ -59,22 +60,21 @@ in
 
 stdenv.mkDerivation rec {
   pname = "espeak-ng";
-  version = "1.52.0";
+  inherit version;
 
   src = fetchFromGitHub {
     owner = "espeak-ng";
     repo = "espeak-ng";
-    tag = version;
-    hash = "sha256-mmh5QPSVD5YQ0j16R+bEL5vcyWLtTNOJ/irBNzWY3ro=";
+    rev = "0d451f8c1c6ae837418b823bd9c4cbc574ea9ff5";
+    hash = "sha256-wpPi+YjSLhsEWfE3KEbL4A7o48qtz9fLRZ/u4xGOM2g=";
```

Comment:
Why is `src` repeated twice in this fine?

Reply:
sorry. I saw this and choose to change as little as possible. Resolved now